PowerUp/app/src/main/res/values/strings.xml

```diff
@@ -133,4 +133,30 @@
     <string name="save_blood_tutorial_3">Try to choose all the correct options before the time runs out!</string>
     <string name="msg1_level_complete">Happy Birthday! You have completed the middle school level!</string>
     <string name="msg2_level_complete">Congratulations! Let\'s now proceed to high school.</string>
+    <string name="f">F</string>
+    <string name="score0">Score: 0</string>
+    <string name="zero">0</string>
+    <!--<string name="_0">0</string>-->
+    <string name="_0">0</string>
+    <!--<string name="_0">0</string>-->
+    <string name="t">T</string>
+    <string name="q">Q</string>
+    <!--<string name="_0">0</string>-->
+    <!--<string name="_0">0</string>-->
+    <string name="pimples">Pimples</string>
+    <string name="bra">Bra</string>
+    <string name="periods">Periods</string>
+    <string name="_skin">Skin</string>
+    <string name="eyes">Eyes</string>
+    <string name="clothes">Clothes</string>
+    <string name="_hair">Hair</string>
+    <!--<string name="_0">0</string>-->
+    <string name="this_is_your_final_avatar_press_continue_to_start_your_journey">This is your final avatar! Press continue to start your Journey...</string>
```

Comment:
The name is too long. Use something short and appropriate.
Could be something like ```final_avatar_message```.